src/it/scala/com/wavesplatform/it/MicroblocksFeeTestSuite.scala

```diff
@@ -0,0 +1,157 @@
+      blockAfterActivation <- notMiner.blockAt(mActivationHeight + 1)
+    } yield {
+
+      balancesBeforeActivation(blockBeforeActivation.generator) shouldBe {
+        initialBalances(blockBeforeActivation.generator) + blockBeforeActivation.fee
+      }
+
+      balancesOnActivation(blockOnActivation.generator) shouldBe {
+        balancesBeforeActivation(blockOnActivation.generator) + blockOnActivation.fee * 0.4
+      }
+
+      balancesAfterActivation(blockAfterActivation.generator) shouldBe {
+        balancesOnActivation(blockAfterActivation.generator) +
+          blockOnActivation.fee * 0.6 + blockAfterActivation.fee * 0.4
+      }
+    }
+
+    Await.result(f, 2.minute)
+  }
+
+  object MicroblocksFeeTestSuite {
+
+    private val dockerConfigs = Docker.NodeConfigs.getConfigList("nodes").asScala
+
+    val NodesCount: Int = 3
+
+    val microblockActivationHeight = 10
+    private val supportedNodes = ConfigFactory.parseString(
+      s"""
+         |waves {
+         |   blockchain {
+         |     custom {
+         |        functionality{
+         |          pre-activated-features = {2=$microblockActivationHeight}
+         |        }
+         |        genesis {
+         |          signature: "gC84PYfvJRdLpUKDXNddTcWmH3wWhhKD4W9d2Z1HY46xkvgAdqoksknXHKzCBe2PEhzmDW49VKxfWeyzoMB4LKi"
+         |          transactions = [
+         |            {recipient: "3Hm3LGoNPmw1VTZ3eRA2pAfeQPhnaBm6YFC", amount: 250000000000000},
+         |            {recipient: "3HPG313x548Z9kJa5XY4LVMLnUuF77chcnG", amount: 250000000000000},
+         |            {recipient: "3HZxhQhpSU4yEGJdGetncnHaiMnGmUusr9s", amount: 250000000000000},
+         |            {recipient: "3HVW7RDYVkcN5xFGBNAUnGirb5KaBSnbUyB", amount: 250000000000000}
+         |          ]
+         |       }
+         |      }
+         |   }
+         |}
+      """.stripMargin
+    )
+    private val notMiner = ConfigFactory.parseString(
+      s"""
+         |waves.miner.enable=no
+         |waves {
+         |   blockchain {
+         |     custom {
+         |        functionality{
+         |          pre-activated-features = {2=$microblockActivationHeight}
+         |        }
+         |        genesis {
+         |          signature: "gC84PYfvJRdLpUKDXNddTcWmH3wWhhKD4W9d2Z1HY46xkvgAdqoksknXHKzCBe2PEhzmDW49VKxfWeyzoMB4LKi"
+         |          transactions = [
+         |            {recipient: "3Hm3LGoNPmw1VTZ3eRA2pAfeQPhnaBm6YFC", amount: 250000000000000},
+         |            {recipient: "3HPG313x548Z9kJa5XY4LVMLnUuF77chcnG", amount: 250000000000000},
+         |            {recipient: "3HZxhQhpSU4yEGJdGetncnHaiMnGmUusr9s", amount: 250000000000000},
+         |            {recipient: "3HVW7RDYVkcN5xFGBNAUnGirb5KaBSnbUyB", amount: 250000000000000}
+         |          ]
+         |       }
+         |      }
+         |   }
+         |}
+      """.stripMargin
+
+    )
+
+
+    val Configs: Seq[Config] = Seq(notMiner.withFallback(dockerConfigs.head)) ++
```

Comment:
`Seq(notMiner.withFallback(dockerConfigs.head), notMiner.withFallback(dockerConfigs(1)), supportedNodes.withFallback(dockerConfigs(2)), supportedNodes.withFallback(dockerConfigs(3)))` is probably what you want